docs/pages/api.rst

```diff
@@ -2254,7 +2320,6 @@ Signature: ``Collection::zip(iterable ...$iterables);``
 .. _Normalizeable: https://github.com/loophp/collection/blob/master/src/Contract/Operation/Normalizeable.php
 .. _Nthable: https://github.com/loophp/collection/blob/master/src/Contract/Operation/Nthable.php
 .. _Nullsyable: https://github.com/loophp/collection/blob/master/src/Contract/Operation/Nullsyable.php
-.. _Onlyable: https://github.com/loophp/collection/blob/master/src/Contract/Operation/Onlyable.php
```

Comment:
this no longer exists, was replaced by `IntersectKeys`

Reply:
Yep, nicely spotted :)